src/App.ts

```diff
@@ -129,7 +129,7 @@ export interface ViewConstraints {
 }
 
 export interface ErrorHandler {
-  (error: CodedError): Promise<void>;
+  (error: CodedError, middlewareArgs: Object): Promise<void>;
```

Comment:
`middlewareArgs`'s type can be a new type that consists of `AnyMiddlewareArgs` + `context`, `logger`, `client` (in other words, `AllMiddlewareArgs` - `next`)